src/components/Tab/index.tsx

```diff
@@ -0,0 +1,21 @@
+import { FC } from 'react'
+
+import { createIcon, getClassNames } from './core/tab.helpers'
+import { IProps } from './core/tab.typings'
+
+export const Tab: FC<IProps> = ({
+  text,
+  color = 'grey-light',
```

Comment:
I think you can change `color` by `selected` for put more logic in component and less somewhere else

Reply:
Do you think about `selected` being a boolean ? Because we have 3 colors on the figma

Reply:
Yes we can play with another props like "disabled" for light grey for example

Reply:
Alright